app/models/balance.rb

```diff
@@ -2,14 +2,28 @@
 class Balance
   include ActiveModel::Model
 
-  Wrapper = Struct.new(:user, :balance, :created_at) do
+  Wrapper = Struct.new(:user, :balance, :pending_balance, :created_at) do
     include ActiveModel::Serialization
   end
 
   def initialize(user)
     @user = user
   end
 
+  def pending_debt
+    Money.new(
+      BalanceQuery.new(user).pending_balances
+        .merge(all_balances) { |_, pending, balance| pending + balance }
+        .select { |_, balance| balance < 0 }
+        .sum { |_, balance| balance },
```

Comment:
What exactly are the arguments you pass to the iteration? On LOC 16 there's 3 of them, what kind of collection is it? There's a lot of `_` being passed around, is there any way to avoid it? Just asking :)

Reply:
On LOC 16 I'm merging two hashes, so the `_` argument is simply a hash key.